modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -464,7 +464,7 @@ const DOMUtils = (doc: Document, settings: Partial<DOMUtilsSettings> = {}): DOMU
   const getStyle = (elm: string | Element | null, name: string, computed?: boolean): string | undefined => {
     const $elm = get(elm);
 
-    if (Type.isNullable($elm) || !NodeType.isElement($elm)) {
+    if (Type.isNullable($elm) || (!NodeType.isHTMLElement($elm) && !NodeType.isSVGElement($elm))) {
```

Comment:
why not continue using `isElement` here?

Reply:
Because Element doesn't have the `style` property. SVGElement and HTMLElement has that property. So any custom element or things like mathml might not have that property.